x-pack/qa/multi-cluster-search-security/legacy-with-basic-license/build.gradle

```diff
@@ -19,7 +19,7 @@ restResources {
 }
 
 // randomise between sniff and proxy modes
-boolean proxyMode = buildParams.random.nextBoolean()
```

Comment:
since we only use this in deprecated TestCluster settings I did not invest to much time to sugar code that api here for now. Ultimately this moves into tests at one point 🤞